src/tests/functional/shared_test_classes/src/base/generate_static_shapes.cpp

```diff
@@ -0,0 +1,157 @@
+                   size_t port,
+                   uint64_t dimMin,
+                   uint64_t dimMax,
+                   int seed) {
+    const auto& partialShape = node->get_input_partial_shape(port);
+    const auto& padsBegin = node->get_pads_begin();
+    const auto& padsEnd = node->get_pads_end();
+    const auto& kernel = node->get_kernel();
+    const auto dilations = ov::Strides(kernel.size(), 1);
+    return pooling::generatePoolingShape(partialShape, dilations, kernel, padsBegin, padsEnd, dimMin, dimMax, seed);
+}
+
+ov::Shape generate(const std::shared_ptr<ov::op::v8::MaxPool>& node,
+                   size_t port,
+                   uint64_t dimMin,
+                   uint64_t dimMax,
+                   int seed) {
+    const auto& partialShape = node->get_input_partial_shape(port);
+    const auto& padsBegin = node->get_pads_begin();
+    const auto& padsEnd = node->get_pads_end();
+    const auto& kernel = node->get_kernel();
+    const auto& dilations =  node->get_dilations();
+    return pooling::generatePoolingShape(partialShape, dilations, kernel, padsBegin, padsEnd, dimMin, dimMax, seed);
+}
+
+ov::Shape generate(const std::shared_ptr<ov::op::v14::MaxPool>& node,
+                   size_t port,
+                   uint64_t dimMin,
+                   uint64_t dimMax,
+                   int seed) {
+    const auto& partialShape = node->get_input_partial_shape(port);
+    const auto& padsBegin = node->get_pads_begin();
+    const auto& padsEnd = node->get_pads_end();
+    const auto& kernel = node->get_kernel();
+    const auto& dilations =  node->get_dilations();
+    return pooling::generatePoolingShape(partialShape, dilations, kernel, padsBegin, padsEnd, dimMin, dimMax, seed);
+}
+
+template<typename T>
+ov::Shape generateStaticShape(const std::shared_ptr<ov::Node>& node,
+                              size_t port,
+                              uint64_t dimMin,
+                              uint64_t dimMax,
+                              int seed) {
+    return generate(ov::as_type_ptr<T>(node), port, dimMin, dimMax, seed);
+}
+} // namespace
+
+StaticShapesMap getStaticShapeMap() {
+    static StaticShapesMap staticShapesMap{
+#define _OPENVINO_OP_REG(NAME, NAMESPACE) {NAMESPACE::NAME::get_type_info_static(), generateStaticShape<NAMESPACE::NAME>},
+
+#include "openvino/opsets/opset1_tbl.hpp"
+#include "openvino/opsets/opset2_tbl.hpp"
+#include "openvino/opsets/opset3_tbl.hpp"
+#include "openvino/opsets/opset4_tbl.hpp"
+#include "openvino/opsets/opset5_tbl.hpp"
+#include "openvino/opsets/opset6_tbl.hpp"
+#include "openvino/opsets/opset7_tbl.hpp"
+#include "openvino/opsets/opset8_tbl.hpp"
+#include "openvino/opsets/opset9_tbl.hpp"
+#include "openvino/opsets/opset10_tbl.hpp"
+#include "openvino/opsets/opset11_tbl.hpp"
+#include "openvino/opsets/opset12_tbl.hpp"
+#include "openvino/opsets/opset13_tbl.hpp"
+#include "openvino/opsets/opset14_tbl.hpp"
+
+#include "ov_ops/opset_private_tbl.hpp"
```

Comment:
We can remove `#include "ov_ops/opset_private_tbl.hpp"`, because we use only public operations in op conformance suite (`ov_subgraphs_dumper` tool used only public opsets to extract operations/subgraphs)

Reply:
Thank you very much for the explanation. Removed.